src/main/resources/db/migration/V4_2__standardtekst.sql

```diff
@@ -0,0 +1,14 @@
+-- ROLLBACK-START
+------------------
+-- DROP TABLE STANDARDTEKST;
+---------------
+-- ROLLBACK-END
+
+CREATE TABLE STANDARDTEKST (
+  id                       SERIAL             PRIMARY KEY,
+  nokkel                   VARCHAR(50)        NOT NULL,
+  gyldig_fom               DATE               NOT NULL,
+  gyldig_til               DATE,
+  tekst                    TEXT               NOT NULL DEFAULT '',
+  UNIQUE (nokkel, gyldig_fom)
```

Comment:
Legg gjerne med:
`created_at               TIMESTAMP          NOT NULL,`

Reply:
I teamet/NAV har vi vel havna på at så mye som mulig av kode og variabler skal være på engelsk, mens domeneord er på norsk. Det er kanskje verdt en diskusjon om vi skal fortsette med det? Og å finne ut hvor grensa for  om noe er et domeneord går, den er ikke alltid like tydelig :/.

Reply:
Hva gjør vi om noen vil oppdatere en tekst to ganger på samme dag?

Reply:
Vil det være behov for å oppdatere tekstene mer enn en gang per dag? Hvis det er rene skrivefeil så antar jeg at det er ok å oppdatere teksten som er gyldig for den aktuelle dagen (dvs i et slikt tilfelle burde det ikke være nødvendig å legge inn en ny rad i databasen for den aktuelle standardteksten, men i stedet oppdatere den som finnes fra før).

Reply:
Behov og behov; det har i hvert fall skjedd tidligere at høyere tekstmakter endrer mening ila en dag, både én og to ganger. Men enig at at skrivefeil og sånt bare er å endre!